pkg/azure/actuator.go

```diff
@@ -64,9 +64,11 @@ func NewActuator(c client.Client) (*Actuator, error) {
 
 	client := newClientWrapper(c)
 	return &Actuator{
-		client:                       client,
-		codec:                        codec,
-		credentialMinterBuilder:      NewAzureCredentialsMinter,
+		client: client,
+		codec:  codec,
+		credentialMinterBuilder: func(logger log.FieldLogger, clientID, clientSecret, tenantID, subscriptionID string) (*AzureCredentialsMinter, error) {
```

Comment:
why introduce the thin wrapper around NewAzureCredentialsMinter()?

Reply:
`NewAzureCredentialsMinter` now takes a `cloudName` parameter. I can either have a wrapper here that closes over the `cloudName` variable, or I can add the cloud name as a field to the actuator. I chose the former.